cpp/csp/cppnodes/statsimpl.h

```diff
@@ -1099,6 +1103,9 @@ void ost_erase( ost<Comparator> &t, double & v )
     auto it = t.find_by_order( rank );
     t.erase( it );
 }
+#else
```

Comment:
Per my benchmarking comment, let's remove all the `#ifdef` statements and fully cut over to the `boost` implementation.